history/characters/wc_easteregg_characters.txt

```diff
@@ -457,4 +457,271 @@ easteregg13={ #Shrek reference
+	584.3.19 = { 
+		birth = yes
+		trait = creature_human
+	}
+	600.1.1 = {
+		set_relation_friend = character:easteregg15
+	}
+	# switches from lvl 4 rogue into paladin around 610, level 4 holy
+}
+
+# Part of Eden Aurorae guild, therefore a landless playable character
+easteregg15 = {
+	name = "Yimo"
+	dna = yimo_dna
+	culture = gnome
+	religion = rationalism
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 5
+	martial = 4
+	stewardship = 4
+	intrigue = 4
+	learning = 20
+	prowess = 15
+	#Personality
+	trait = compassionate
+	trait = generous
+	trait = diligent
+	#Education
+	trait = education_learning_4
+	#Other
+	trait = intellect_good_1
+	trait = magic_good_3
+	536.5.10 = { 
+		birth = yes
+		trait = creature_gnome
+	}
+	# level 4 frost mage
+}
+
+# Tauren mercenary, therefore a landless playable character
+easteregg16 = {
+	name = "Ball"
+	#dna = ball_dna
+	culture = tauren
+	religion = earth_mother_worship
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 4
+	martial = 18
+	stewardship = 3
+	intrigue = 3
+	learning = 4
+	prowess = 20
+	#Personality
+	trait = brave
+	trait = stubborn
+	trait = content
+	#Education
+	trait = education_martial_3
+	#Other
+	trait = physique_good_1
+	547.8.19 = { 
+		birth = yes
+		trait = creature_tauren
+	}
+	# level 4 warrior
+}
+
+# Tauren mercenary, therefore a landless playable character
+easteregg17 = {
+	name = "Jubai"
+	#dna = jubai_dna
+	culture = tauren
+	religion = earth_mother_worship
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 4
+	martial = 22
+	stewardship = 4
+	intrigue = 3
+	learning = 3
+	prowess = 20
+	#Personality
+	trait = brave
+	trait = forgiving
+	trait = honest
+	#Education
+	trait = education_learning_4
+	#Other
+	trait = magic_good_3
+	506.3.10 = { 
+		birth = yes
+		trait = creature_tauren
+	}
+	583.1.1 = {
+		give_nickname = nick_the_ancient
+	}
+	# level 4 druid
+}
+
+# Dwarven officer of Ironforge
+easteregg18 = {
+	name = "Serphentos"
+	#dna = serphentos_dna
+	culture = bronzebeard
+	religion = khazism
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 3
+	martial = 20
+	stewardship = 8
+	intrigue = 5
+	learning = 3
+	prowess = 25
+	#Personality
+	trait = vengeful
+	trait = honest
+	trait = brave
+	#Education
+	trait = education_martial_4
+	#Other
+	trait = physique_good_2
+	trait = faith_warrior
+	trait = lifestyle_blademaster
+	377.8.24 = { 
+		birth = yes
+		trait = creature_dwarf
+	}
+	511.7.15 = {
+		employer = 17001 # Magni [4200]
+		give_council_position = councillor_marshal
+	}
+	# level 5 warrior
+}
+
+# Sentinel captain of the Alliance
+easteregg19 = {
+	name = "Monóri"
+	#dna = monori_dna
+	dynasty = easteregg_dyn11 # Silverleaf
+	culture = night_elf
+	religion = kaldorei_religion
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 10
+	martial = 15
+	stewardship = 5
+	intrigue = 3
+	learning = 4
+	prowess = 25
+	#Personality
+	trait = compassionate
+	trait = generous
+	trait = diligent
+	#Education
+	trait = education_martial_5
```

Comment:
I think level 5 martial is too much for an easter egg character, especially considering that most lore characters like Maiev and Broll Bearmantle don't exceed level 4.

Reply:
rebalanced